analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs

```diff
@@ -515,7 +515,10 @@ public static void VerifyNoExceptionThrown(string path, DiagnosticAnalyzer[] dia
                                                                         IEnumerable<ParseOptions> options = null)
             where TMessage : IMessage<TMessage>, new()
         {
-            var solutionBuilder = SolutionBuilder.CreateSolutionFromPaths(paths);
+            var solutionBuilder = SolutionBuilder.Create()
+                .AddProject(AnalyzerLanguage.FromPath(paths.First()))
+                .AddDocuments(paths)
+                .Solution;
```

Comment:
This section now lost a little bit of path validation. It's not an issue as it will be completely removed in #5271